packages/blade/src/components/Divider/Divider.tsx

```diff
@@ -94,6 +95,9 @@ const Divider = ({
       {...borderColor}
       {...metaAttribute({ name: MetaConstants.Divider, testID })}
       {...getStyledProps(styledProps)}
+      {...makeAccessible({
```

Comment:
I see now why we are getting accessible={true} 

Let's explicitly override this prop in RN to be false. Otherise dividers will unnecessarily receive focus.

Reply:
ahh got it. I removed the makeAccessible call itself from react native.